src/globus_sdk/login_flows/local_server_login_flow_manager/local_server_login_flow_manager.py

```diff
@@ -132,20 +116,25 @@ def for_globus_app(
         cls, app_name: str, login_client: AuthLoginClient, config: GlobusAppConfig
     ) -> LocalServerLoginFlowManager:
         """
-        Create a ``LocalServerLoginFlowManager`` for a given ``GlobusAppConfig``.
-
-        :param app_name: The name of the app to use for prefilling the named grant,.
-        :param login_client: The ``AuthLoginClient`` to use to drive Globus Auth flows.
-        :param config: A ``GlobusAppConfig`` to configure the login flow.
-        :returns: A ``LocalServerLoginFlowManager`` instance.
-        :raises: GlobusSDKUsageError if a custom login_redirect_uri is defined in
-            the config.
+        Create a ``LocalServerLoginFlowManager`` for use in a GlobusApp.
+
+        :param app_name: The name of the app. Will be prefilled in native auth flows.
+        :param login_client: A client used to make Globus Auth API calls.
+        :param config: A GlobusApp-bounded object used to configure login flow manager.
+        :raises: GlobusSDKUsageError if app config is incompatible with the manager.
```

Comment:
Tagging this as an example to change but there may be others (or, we can do a mass-update after this merges).

```suggestion
        :raises GlobusSDKUsageError: if app config is incompatible with the manager.
```